Update Java SDK docs to 3.0 (#2239)
getsentry/sentry-docs: src/platforms/java/common/legacy/index.mdx

---
title: Legacy SDK (1.7)
sidebar_order: 2000
---

> This section covers legacy Sentry SDK for Java (1.7.x). Legacy SDK is not under active development anymore, and we **strongly encourage** updating to the most recent Sentry SDK.

Reviewer: ```suggestion
This section covers the legacy Sentry SDK for Java (1.7.x), which is no longer under active development. We **strongly encourage** updating to [the updated Java SDK](/platforms/java/).
```
